llarp/crypto/types.hpp

```diff
@@ -1,10 +1,13 @@
 #pragma once
 
 #include "constants.hpp"
+
 #include <llarp/router_id.hpp>
 #include <llarp/util/aligned.hpp>
-#include <llarp/util/types.hpp>
 #include <llarp/util/fs.hpp>
+#include <llarp/util/types.hpp>
+
+#include <sodium.h>
```

Comment:
This is putting sodium headers in nearly every compilation unit across the codebase, whereas sodium is meant to be just an internal implementation detail of the .cpp.